.github/workflows/daily.yml

```
@@ -0,0 +1,243 @@
+name: Daily ScalarDB cluster test
```

```suggestion
name: Daily ScalarDB Cluster test
```

I think, basically, we capitalize `C` of `cluster` because the term `ScalarDB Cluster` is a product (or component?) name.